ecommerce-product-page-main/styled.css

```diff
@@ -116,37 +118,34 @@ body{
     }
 
     .dropdown{
-
         visibility: hidden;
     }
 
 
-        .button2{
-            margin-top: 20px;
-        }
 }
 
+.containflex{
+    display: flex;
+}
 
 
 .container{
-
+    padding-left: 20px;
+    padding-right: 20px;
     display:flex;
-    padding-bottom: 15px;
-    padding-top: 30px;
+    justify-content: space-between;

```

Comment:
What is this flex property for exactly again?

Reply:
it will create it space between the element inside the tag this was applied to